crates/astria-sequencer-relayer/src/relayer/write/conversion.rs

```diff
@@ -78,11 +86,31 @@ pub(super) fn convert(block: SequencerBlock) -> eyre::Result<Converted> {
             celestia_namespace: namespace,
             sequencer_rollup_id: blob.rollup_id(),
         };
-        let blob = Blob::new(namespace, blob.into_raw().encode_to_vec())
+        let raw_blob = blob.into_raw().encode_to_vec();
+        total_data_uncompressed_size += raw_blob.len();
+        let compressed_blob = brotli_compressed_bytes(&raw_blob)
+            .wrap_err_with(|| format!("failed compressing rollup `{rollup_id}`"))?;
+        total_data_compressed_size += compressed_blob.len();
+        let blob = Blob::new(namespace, compressed_blob)
             .wrap_err_with(|| format!("failed creating blob for rollup `{rollup_id}`"))?;
         blobs.push(blob);
         rollups.push(info);
     }
+
+    let compression_ratio = total_data_uncompressed_size / total_data_compressed_size;
```

Comment:
Might be nice to just do the casts to f64 here since we need to cast the result anyway.  Would give a slightly more accurate value I guess, especially if compression is not too effective.
```suggestion
    let compression_ratio = total_data_uncompressed_size as f64 / total_data_compressed_size as f64;
```

Reply:
Good catch, I missed that completely.